templates/indicators/result_form_common_js.html

```diff
@@ -84,7 +84,7 @@
                         // Close modal
                         $('#indicator_results_div').modal('hide');
                         $('#indicator_results_div').trigger('save.tola.result_form');
-                        PNotify.success("Result deleted")
+                        window.unified_success_message("Result deleted");
```

Comment:
Wouldn't hurt to add translator comment, just in case we ask translators to do an overall review at some point.